dln/operator.py

```diff
@@ -16,21 +18,20 @@
 from termcolor import colored
 import yaml
 
+logging.getLogger("openai").setLevel(logging.WARNING)
 
-openai.util.logger.setLevel(logging.WARNING)
-
+class InvalidRequestError(Exception):
+    """Exception raised for invalid request errors."""
+    pass
 
 def _retry_request(min_wait=4, max_wait=10, max_attempts=100):
     return retry(
         reraise=True,
         stop=stop_after_attempt(max_attempts),
         wait=wait_exponential(multiplier=1, min=min_wait, max=max_wait),
         retry=(
-            retry_if_exception_type(openai.error.Timeout)
-            | retry_if_exception_type(openai.error.APIError)
-            | retry_if_exception_type(openai.error.APIConnectionError)
-            | retry_if_exception_type(openai.error.RateLimitError)
-            | retry_if_exception_type(openai.error.ServiceUnavailableError)
+            retry_if_exception_type(openai.Timeout)
+            | retry_if_exception_type(openai.APIError)
```

Comment:
Why discard the `APIConnectionError` and `RateLimitError`?

Reply:
Good catch, reverted.